client/src/utils/consts.js

```diff
@@ -0,0 +1,9 @@
+export const levelEnum = {
+  ADMIN: 4,
```

Comment:
Are these numbers needed anywhere or could we redefine this enum to match keys to values (e.g. `ADMIN: "ADMIN"`)

Reply:
They're used for the ranged level comparison like on line 37 of `client/src/components/navbar/Navbar.js`. I think numbers is fine because it's easier to say `if userLevel > minAccessLevel`